netlify.toml

```diff
@@ -1,8 +1,14 @@
 # This should be used for docs also
+[[plugins]]
+  package = "@netlify/plugin-csp-nonce"
+  [plugins.inputs]
+    reportOnly = true
+    path = "/*"
+
 [[headers]]
   for = "/*"
   [headers.values]
-    Content-Security-Policy = "default-src https:; script-src https: 'unsafe-inline'; style-src https: 'unsafe-inline'; img-src https: data: blob:; frame-ancestors *"
+    Content-Security-Policy = "default-src 'self'; script-src 'self' 'unsafe-inline'; style-src https:; img-src https: data: blob:; frame-ancestors *; object-src 'none'; connect-src https:; require-trusted-types-for 'script'"
```

Comment:
From the SvelteKit docs:
![image](https://github.com/rilldata/rill/assets/14206386/08150c34-ba07-4880-95f6-1ce9013ab6ab)

Reply:
Given we're building a static site, we should use hashes not nonces. Leaving the blog post I shared with you here for us to come back to later: https://dev.to/askrodney/sveltekit-content-security-policy-csp-for-xss-protection-589k